QuEST/include/QuEST.h

```diff
@@ -1662,6 +1662,43 @@ int measureWithStats(Qureg qureg, int measureQubit, qreal *outcomeProb);
  */
 Complex calcInnerProduct(Qureg bra, Qureg ket);
 
+/** If both input states \p a and \p b are equal-size density matrices
+ * then computes the Hilbert-Schmidt scalar product between \p a and \p b, 
+ * which is equialent to the Frobenius inner product of matrices.
```

Comment:
typo: equialent

Reply:
Addressed this in acf7f95214544204939c8267381218a72ec98096.

Reply:
Haha actually you missed it! :)